app/assets/javascripts/sufia/featured_works.js

```diff
@@ -31,20 +31,37 @@ function dragAndDrop(selector) {
 }
 
 Blacklight.onLoad(function() {
-  $('a[data-behavior="feature"]').on('click', function(evt) {
+  $('#content-wrapper').on('click', 'a[data-behavior="feature"]', function(evt) {
     evt.preventDefault();
+    evt.stopPropagation();
```

Comment:
We're not sure why. Without it, it runs the JS, fixes the link, and then redirects to the JSON despite preventDefault.